n2ws/README.md

```diff
@@ -0,0 +1,86 @@
+# N2WS Backup & Recovery
+
+## Overview
+
+N2WS Backup & Recovery (CPM), known as N2WS, is an enterprise-class backup, 
+recovery and disaster recovery solution for the Amazon Web Services (AWS). 
+Designed from the ground up to support AWS, N2WS uses cloud native technologies 
+(e.g. EBS snapshots) to provide unmatched backup and, more importantly, restore capabilities in AWS.
+
+N2WS Backup and Recovery Instance supports the monitoring of backups, DR, copy to S3, alerts 
+and more by Datadog monitoring service. 
+It will allow users to monitor and analyse the N2WS Backup and Recovery Dashboard metrics.
+
+## Setup
+
+### Installation
+
+Please perform the following steps in order to activate the service and to monitor your N2WS instance:
+
+1.	Setup Datadog Account 
+Visit Datadog to: https://www.datadoghq.com/pricing/ and setup an account that fits your scale.
+
+
+2.	Install Python Integration
+Login to Datadog and go to Integrations -> Integrations
+Search for ‘Python’ and install it
+
+
+3.	Enable Datadog support on N2WS Instance
+Connect to your N2WS Backup and Recovery Instance with SSH Client, type “sudo su” and add the following lines to /cpmdata/conf/cpmserver.cfg
+[external_monitoring]
+enabled=True
+run ‘service apache2 restart’
+
+
+4.	Install Datadog Agent on N2WS Instance
+Login to Datadog and go to Integrations -> Agent -> Ubuntu
+Copy the agent ‘easy one-step install’ command 
+Connect to your N2WS Backup and Recovery Instance with SSH Client, type “sudo su” and  run the agent Install command.
```

Comment:
Instead of walking through that whole thing, let's just say:

Connect to your N2WS Backup and Recovery instance with SSH and run the following command to install the Datadog Agent. You may require `sudo` privileges to perform this action: `<insert command here>`

Reply:
Every DataDog account has a different command, that is why we can't insert a command here.